discovery-common/src/main/java/app/metatron/discovery/extension/dataconnection/jdbc/accessor/AbstractJdbcDataAccessor.java

```diff
@@ -213,7 +213,14 @@ public void useDatabase(String catalog, String database) {
       // 1. TABLE_SCHEM String => schema name
       // 2. TABLE_CATALOG String => catalog name (may be null)
       while (rs.next()) {
-        dataBaseNames.add(rs.getString(1));
```

Comment:
@YooYoungmo 
The DatabaseMetadata object has the getSchemas (catalog, schemaPattern) function.
What do you think of the modified code below?
```
      String schemaPatternParam = schemaPattern;
      if(StringUtils.isNotEmpty(schemaPattern)){
        schemaPatternParam = "%" + schemaPattern + "%";
      }
      conn = this.getConnection();
      rs = conn.getMetaData().getSchemas(catalog, schemaPatternParam);
```

Reply:
@ufoscw It's a good idea. I changed the code.